src/Uno.UI.Runtime.Skia.Wpf/UI/Controls/WpfWindowWrapper.cs

```diff
@@ -121,6 +121,7 @@ private void UpdateIsVisible()
 
 	protected override IDisposable ApplyFullScreenPresenter()
 	{
+		_isFullScreen = true;
```

Comment:
do we need more tests if possible here ? or a manual one?

Reply:
There are already manual tests for this.